geschaeftsordnung-vollversammlung.md

```diff
@@ -0,0 +1,82 @@
+# Geschäftsordnung der Vollversammlung der Fachschaft Digital Engineering
+
+Diese Geschäftsordnung wurde am xx.xx.xxxx von der Vollversammlung beschlossen.
+
+## § 1 Geltungsbereich
+
+Die Geschäftsordnung gilt für die Vollversammlung der Fachschaft Digital Engineering an der Universität Potsdam.
+
+## § 2 Tagesordnung
+
+(1) Die Vollversammlung folgt einer Tagesordnung.
+
+(2) Die Tagesordnung muss Teil der fristgerechten Einladung einer Vollversammlung sein.
+
+(3) Bis 14 Tage vor einer Vollversammlung können die einladenden Personen gestellte Anträge in die Tagesordnung aufnehmen.
+
+(4) Bis 7 Tage vor einer Vollversammlung muss die finale Tagesordnung auf dem gleichen Weg wie die Einladung zur Vollversammlung bekanntgegeben werden.
+
+## § 3 Versammlungsleitung
+
+(1) Die Vollversammlung wird von den einladenden Personen geleitet.
+
+(2) Die Versammlungsleitung kann für einzelne Tagesordnungspunkte an andere Mitglieder der Fachschaft übergeben werden.
+
+(3) Die Versammlungsleitung kann bei Bedarf eine Redeliste führen und/oder die Redezeit begrenzen.
+
+(4) Die Versammlungsleitung entscheidet, auf welche Weise Stimmen gezählt oder geschätzt und ob sie anschließend protokolliert werden.
+
+## § 4 Protokolle
+
+(1) Die Vollversammlung dokumentiert ihre Beschlüsse in einem Protokoll.
+
+(2) Die Versammlungsleitung beauftragt ein Mitglied der Fachschaft mit der Protokollführung.
+
+(3) Das Protokoll wird der Fachschaft spätestens 7 Tage nach der Vollversammlung zugänglich gemacht.
+
+## § 5 Anträge
+
+(1) Anträge sind schriftlich zu verfassen und innerhalb von 14 Tagen nach Zugang der Einladung zur Vollversammlung bei den einladenden Personen einzureichen.
+
+(2) Die einladenden Personen entscheiden über die Aufnahme des Antrags in die Tagesordnung.
+
+(3) Anträge können vor der Behandlung durch die antragstellende Person zurückgezogen werden.
+
+## § 6 Änderungsanträge
+
+(1) Änderungsanträge werden vor der Abstimmung des jeweiligen Antrags behandelt.
```

Comment:
Was ist mit Änderungsanträgen gegen Änderungsanträgen?

Blödes Beispiel, aber zur Visualisierung:
Antrag: "Die Fachschaft wird 500 € für X ausgeben",
Änderungsantrag: "500 € wird durch 1000 € ersetzt", 
Änderungsänderungsantrag: "1000 € wird durch 'eine beliebige, vom FSR festgelegte Summe' ersetzt"...


Die sind eigentlich immer unnötig, v.a. da man ja einfach über einen neuen Ä-Antrag entscheiden kann. Daher würde ich die ausschließen.

Reply:
Dieses Konstrukt wäre in der Theorie vermutlich möglich, in der Praxis würde man aber so verfahren, wie du am Ende beschreibst.
Der Satz soll ausdrücken, dass Man nicht erst über einen Antrag und dann über einen darauf bezogenen Änderungsantrag abstimmt.